components/react/views/Connected.tsx

```diff
@@ -80,7 +87,9 @@ export const Connected = ({
         <div className="flex items-center ">
           <ProfileAvatar walletAddress={address ?? ''} size={60} />
           <div className="ml-4">
-            <p className="text-lg font-semibold">{username || 'Anonymous'}</p>
+            <p className="text-lg font-semibold">
+              {specialUsernames[name.toUpperCase()] || username || 'Anonymous'}
```

Comment:
```suggestion
              {specialUsernames[name.toUpperCase() as keyof typeof specialUsernames] || username || 'Anonymous'}
```

`bun run build` fails otherwise.